lib/prelude.dx

```diff
@@ -1243,6 +1266,12 @@ def split_v {a} : Iso a ({|} | a) =
 def slice_fields {a b c v} [Ix b] (iso: Iso ({|} | a) (b | c)) (tab: a=>v) : b=>v =
   reindex (build_with $ split_v &>> iso) tab
 
+-- TODO: replace `slice` with this?
+def post_slice {a n} (xs:n=>a) (start:Post n) (end:Post n) : List a =
+  slice_size = ordinal end - ordinal start
```

Comment:
What will this do if end < start?  Should this code include an explicit check?

Reply:
Good point! Parameterizing `Fin` by a `Nat` instead of an `Int` would have caught that.

Reply:
I think it will be fine, because `Fin n` for negative `n` is an empty index set.